pipeline/pipeline-build.yml

```diff
@@ -0,0 +1,54 @@
+resources:
```

Comment:
What is this used for?

Reply:
It's a build NOT deploy pipeline.  Easier for testing that part of the process without a deploy.